backend/core/domains/contracts/client_views.py

```
@@ -0,0 +1,322 @@
+            'signature_timestamp': request.data.get('signature_timestamp', ''),
+            'screen_resolution': request.data.get('screen_resolution', ''),
+        }
+        
+        # Validate signature data
+        serializer = ContractSignatureCreateSerializer(data=signature_data)
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        
+        try:
+            # Create the signature
+            signature = ContractSignatureService.add_signature(
```

Ruff (0.13.3)

Local variable `signature` is assigned to but never used

Remove assignment to unused variable `signature`

(F841)